apps/web/app/(app)/mail/page.tsx

```diff
@@ -25,9 +25,15 @@ export default function Mail({
     {
       keepPreviousData: true,
       dedupingInterval: 1_000,
+      revalidateOnFocus: false,
     },
   );
 
+  // States to handle more emails for the load more feature
+  const [allThreads, setAllThreads] = useState(data?.threads || []);
+  const [nextPageTokenId, setNextPageTokenId] = useState(data?.nextPageToken);
+  const [isLoadMoreLoading, setIsLoadMoreLoading] = useState(false);
```

Comment:
Can use `useSWRInfinite` for potentially cleaner solution here:
https://swr.vercel.app/docs/pagination#example-1-index-based-paginated-api

Reply:
Done